versions-prod.cfg

```diff
@@ -81,21 +81,3 @@ zope.testbrowser = 6.0
 zope.testing = 5.0.1
 zope.traversing = 5.0
 zope.viewlet = 5.0
-
-[versions:python38]
-# Chameleon >= 4.5 requires Python 3.9
-Chameleon = 4.4.4
```

Comment:
This should still be there even though we currently do not use version 4.5+ otherwise we have to find out it again when updating.

Reply:
I agree, it just so happens that Chameleon 4.4.4 is the latest version that lets the tests run so the pins would be the same for Python 3.8 and everything else. I put it back in.

There is a newer release of Chameleon but the tests fail. (different issue for a different day)